packages/ext-angular-modern-kitchensink/src/app/app.module.ts

```diff
@@ -107,6 +101,9 @@ const routes: ExtAngularRoutes = [
 	{ path: 'components/progressbar', component: ProgressBarComponent, text: 'Progress Bar', iconCls: 'x-fa fa-calendar', xtype: 'homeview', leaf: true },
 	{ path: 'components/titlebar', component: TitleBarComponent, text: 'Title Bar', iconCls: 'x-fa fa-calendar', xtype: 'homeview', leaf: true },
 	{ path: 'components/toolbar', component: ToolBarComponent, text: 'Tool Bar', iconCls: 'x-fa fa-calendar', xtype: 'homeview', leaf: true },
+	{ path: ':a/:b/:c', component: LandingpageComponent, text: 'LandingPage', iconCls: 'x-fa fa-calendar', xtype: 'homeview', leaf: true },
```

Comment:
Is this for testing?

Reply:
This is a temporary workaround to get the router to respond to arbitrarily deep paths until I figure out the proper way to do it.